src/challenges.js

```diff
@@ -19,11 +25,17 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  function increment() {
+    return count++;
```

Comment:
You should have it so that it returns `++count` . When you have count++ it will save the count before it iterates which is something that we don't want!